libsolidity/interface/StandardCompiler.cpp

```diff
@@ -1588,7 +1588,7 @@ Json StandardCompiler::compileSolidity(StandardCompiler::InputsAndSettings _inpu
 		if (!contractData.empty())
 		{
 			if (!contractsOutput.contains(file))
-				contractsOutput[file] = Json::object();
+				contractsOutput[file];
```

Comment:
This looks funny now :-). But I guess this ist still in fact equivalent to ``= Json::object();``? I.e. the index access will create a default-initialized ``Json`` and then adding the `name` field below forces it into an "object"?

Reply:
Yeah that looks funny :) However, those cases I didn't wanted to remove :D